pyatv/support/opack.py

```diff
@@ -15,6 +14,8 @@
 
 _SIZED_INT_TYPES: Dict[int, Type] = {}
 
+_OPACK_TIME_OFFSET: float = 978307200.0
```

Comment:
Can you add a comment (and possibly rename the constant) making it clear that this is a conversion from offset between unix time and mach absolute time? This is a very magic constant.